blocks/identity-block/components/FormInputField/index.story.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
```

Comment:
We want to make sure this story file is not published with the block

Reply:
Ah, so this is why we've been putting the story file in the root?  I was curious about this.  Is there a way that we can resolve this or do I move all this into the root level and split it apart somehow in the story configuration?

Reply:
We can add an npmignore to ignore story files just like we do for the shared styles folder

Reply:
In the identity-block/components?
![image](https://user-images.githubusercontent.com/2287238/130140444-0ae7ee33-0c65-4d03-ba9e-ce2e36086f62.png)

Reply:
Yes, that should work

Reply:
To test you can run `npm pack` from the the directory `blocks/identity-block` and it will generate a zip file with what would be published for the package - it will show you in the console output all the files that are included

Reply:
npmignore is now ncluded